testsAT/src/test/java/com/stratio/tests/ATCassandraXDTest.java

```diff
@@ -34,7 +34,8 @@
 //Indicar feature
 @CucumberOptions(features = { "src/test/resources/features/Cassandra/CassandraSelectSimple.feature",
```

Comment:
Just for the next PR:
You can use features = "src/test/resources/features/Cassandra/"
and got all .features in this folder...